docs/js/graphics-element/api/graphics-api.js

```diff
@@ -686,7 +686,7 @@ class GraphicsAPI extends BaseAPI {
    * convenient grid drawing function
    */
   drawGrid(division = 20) {
-    let w = this.panelWidth ?? this.width;
+    let w = this.panelWidth || this.width;
```

Comment:
this is new ES syntax that is supported by both the current LTS version of Node (v14) as well as the current stable version (v15).

Reply:
Didn't know that. Is there a way to exclude last commit from this pull ?

Reply:
yep: in your terminal type `git reset HEAD~1`, which rolls you back 1 commit (using ~2 rolls back 2 commits, etc. And it's always relative to where you are now, so `git reset HEAD~1` followed by another `git reset HEAD~1` has the same effect as `git reset HEAD~2`).

You can then "force push" that rollback up using `git push origin master -f` where the `-f` tells git that you're okay with the fact that this is going to change the commit sequence in a way that doesn't follow the regular "new commit on top of new commit" procedure.

Reply:
Undone.